samples/core/tutorials/estimators/wide.ipynb

```diff
@@ -0,0 +1,1273 @@
+        "    <img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" /><span>Run in Google Colab</span></a>  \n",
+        "</td><td>\n",
+        "<a target=\"_blank\"  href=\"https://github.com/tensorflow/models/blob/master/samples/core/tutorials/estimators/wide.ipynb\"><img width=32px src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" /><span>View source on GitHub</span></a></td></table>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "77aETSYDcdoK",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In this tutorial, we will use the `tf.estimator` API in TensorFlow to solve a\n",
+        "binary classification problem: Given census data about a person such as age,\n",
+        "education, marital status, and occupation (the features), we will try to predict\n",
+        "whether or not the person earns more than 50,000 dollars a year (the target\n",
+        "label). We will train a **logistic regression** model, and given an individual's\n",
+        "information our model will output a number between 0 and 1, which can be\n",
+        "interpreted as the probability that the individual has an annual income of over\n",
+        "50,000 dollars.\n",
+        "\n",
+        "## Setup\n",
+        "\n",
+        "To try the code for this tutorial:\n",
+        "\n",
+        "[Install TensorFlow](tensorlfow.org/install) if you haven't already.\n",
+        "\n",
+        "Next import the relavant packages:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "NQgONe5ecYvE",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "import tensorflow as tf\n",
+        "import tensorflow.feature_column as fc \n",
+        "tf.enable_eager_execution()\n",
+        "\n",
+        "import os\n",
+        "import sys\n",
+        "from IPython.display import clear_output"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "-MPr95UccYvL",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Download the [tutorial code from github](https://github.com/tensorflow/models/tree/master/official/wide_deep/),\n",
+        " add the root directory to your python path, and jump to the `wide_deep` directory:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "yVvFyhnkcYvL",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "if \"wide_deep\" not in os.getcwd():\n",
+        "    ! git clone --depth 1 https://github.com/tensorflow/models\n",
+        "    models_path = os.path.join(os.getcwd(), 'models')\n",
+        "    sys.path.append(models_path)   \n",
+        "    os.environ['PYTHONPATH'] += os.pathsep+models_path\n",
+        "    os.chdir(\"models/official/wide_deep\")"
```

Comment:
nit: This CD isn't strictly necessary, and, since we take the PYTHONPATH route in the official models in general, might lead to some confusion/future errors. The code below would just have to be updated to `from official.wide_deep import ...` instead of just `import ...`

Reply:
Fixed.

It's much cleaner like that. Thanks.